addons/payment_xendit/static/src/js/payment_form.js

```diff
@@ -138,7 +138,17 @@ paymentForm.include({
      */
     _xenditHandleResponse(err, token, processingValues) {
         if (err) {
-            this._displayErrorDialog(_t("Payment processing failed"), err.message);
+            let err_message = err.message;
+
+            // If the error is API_VALIDATION_ERROR: there's an invalid input
+            if (err.error_code === 'API_VALIDATION_ERROR') {
```

Comment:
Are you sure that only `API_VALIDATION_ERROR`   can have `err.errors` ? I have not found info regarding the error object in the documentation.

Reply:
Yes, there's no documentation of what the error object look like for all errors. That's the reason why I'm restricting it to specific error_code "API_VALIDATION_ERROR", which is trigerred when input is in the wrong format